app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -292,6 +314,13 @@
                         track_label: popular_link[:href],
                         track_dimension: popular_link[:label],
                         track_dimension_index: "29",
+                        ga4_link: {
```

Comment:
Is there a test that can be written to check these attributes are set correctly?

Reply:
@andysellick Good spot, I've written a ruby test now :+1: